src/Tribe/Main.php

```diff
@@ -2642,6 +2642,7 @@ public function dateToTimeStamp( $date, $hour, $minute, $meridian ) {
 		 * @todo remove - unused
 		 */
 		protected function dateHelper( $date ) {
+			_deprecated_function( __FILE__, '3.11', 'date' );
```

Comment:
Yeah, though we hadn't thrown deprecation notices for it before, so it is _possible_ that someone is using one of those functions.  With the notice in place, we can trash the functions in a future release.